integration_tests/e2e/reviewPlan/reviewPlan.cy.ts

```diff
@@ -42,8 +42,7 @@ context(`Review a prisoner's plan`, () => {
     cy.visit(`/plan/${prisonNumber}/review`)
 
     // Then
-    Page.verifyOnPage(WhoCompletedReviewPage) //
-      .hasCorrectBackLink()
```

Comment:
I couldn't really see what value the new `hasCorrectBackLink` assertion had (what is "correct" about it?); so I've removed it